nipype/algorithms/modelgen.py

```diff
@@ -176,6 +176,7 @@ def bids_gen_info(bids_event_files,
             f_events = csv.DictReader(f, skipinitialspace=True, delimiter='\t')
             events = [{k: v for k, v in row.items()} for row in f_events]
         conditions = list(set([i[condition_column] for i in events]))
```

Comment:
Just to condense this into one line:

```suggestion
        conditions = sorted(set([i[condition_column] for i in events]))
```